src/Sylius/Bundle/ReportBundle/Controller/ReportController.php

```diff
@@ -70,6 +72,8 @@ public function embedAction(Request $request, $report, array $configuration = ar
         }
 
         $configuration = $request->query->get('configuration', $configuration);
+        $configuration['baseCurrency']= $currencyProvider->getBaseCurrency();
```

Comment:
```
array:9 [▼
  "start" => DateTime {#1679 ▶}
  "end" => DateTime {#1696 ▶}
  "period" => "month"
  "empty_records" => false
  "baseCurrency" => Currency {#3310 ▶}
  "interval" => "P1M"
  "periodFormat" => "%m"
  "presentationFormat" => "F Y"
  "groupBy" => array:2 [▶]
]
```
seems to be mixed, I will leave it like this, but yeah has to have consistency at some point